apps/snitch_core/lib/core/domain/order/transitions.ex

```diff
@@ -37,15 +37,11 @@ defmodule Snitch.Domain.Order.Transitions do
           }
         } = context
       ) do
-    changeset =
-      order
-      |> Repo.preload([:billing_address, :shipping_address])
-      |> Order.partial_update_changeset(%{
-        billing_address: billing,
-        shipping_address: shipping
-      })
-
-    case Repo.update(changeset) do
+    order
+    |> Repo.preload([:billing_address, :shipping_address])
```

Comment:
Since billing and shipping address are now maps, it does not make sense to preload them.